python/paddle/v2/fluid/distribute_transpiler.py

```diff
@@ -303,8 +307,8 @@ def _is_op_on_pserver(self, endpoint, all_ops, idx):
                 return True
             else:
                 for n in param_names:
-                    if n.startswith(op.inputs["Param"].name+".block") and \
-                       n != op.inputs["Param"].name:
+                    if same_or_split_var(n, op.inputs[
```

Comment:
Won't the logic remain same if we do not change this line?

Reply:
Thanks! Wanted to consolidate the code.